admin-console/src/main/webapp/src/components/SystemBehaviorGroups/BehaviorGroupTable.tsx

```diff
@@ -0,0 +1,170 @@
+
+    const handleDelete = React.useCallback(async () => {
+        setShowDeleteModal(false);
+        const deleteBehaviorGroup = deleteBehaviorGroupMutation.mutate;
+        const response = await deleteBehaviorGroup(systemBehaviorGroup.id);
+        if (response.error) {
+            return false;
+        }
+
+        return true;
+    }, [ deleteBehaviorGroupMutation.mutate, systemBehaviorGroup.id ]);
+
+    const onClose = () => {
+        setShowModal(false);
+        getBehaviorGroups.query;
+    };
+
+    const onDeleteClose = () => {
+        setShowDeleteModal(false);
+        getBehaviorGroups.query;
+    };
+
+    if (getBehaviorGroups.loading) {
+        return <Spinner />;
+    }
+
+    if (getBehaviorGroups.payload?.status !== 200) {
+        return <span>Error while loading sysem behavior groups: {getBehaviorGroups.errorObject.toString()}</span>;
+    }
+
+    return (
+        <React.Fragment>
+            <PageSection>
+                <Title headingLevel='h1'>
+                    <Breadcrumb>
+                        <BreadcrumbItem target='#'> Bundles </BreadcrumbItem>
+                        <BreadcrumbItem target='#' >{ (getBundles.loading || getBundles.payload?.status !== 200)
+                            ? <Spinner /> : getBundles.payload.value.displayName }
+                        </BreadcrumbItem>
+                        <BreadcrumbItem target='#'> System Behavior Groups </BreadcrumbItem>
+                    </Breadcrumb>
+                </Title>
+                <TableComposable aria-label="System behavior groups table">
+                    <Thead>
+                        <Toolbar>
+                            <ToolbarContent>
+                                <ToolbarItem>
+                                    <Button variant='primary' type='button' onClick={ createBehaviorGroup }> Create new group </Button>
+                                    <CreateEditBehaviorGroupModal
+                                        isEdit={ isEdit }
+                                        isOpen={ isOpen }
+                                        initialSystemBehaviorGroup={ systemBehaviorGroup }
+                                        showModal={ showModal }
+                                        onClose={ onClose }
+                                        onSubmit={ handleSubmit }
+                                        isLoading={ false }
+                                    />
+                                    <DeleteBehaviorGroupModal
+                                        onDelete={ handleDelete }
+                                        bundleName={ bundle?.displayName }
+                                        systemBehaviorGroupName={ systemBehaviorGroup.displayName }
+                                        isOpen={ showDeleteModal }
+                                        onClose={ onDeleteClose }
+                                    />
```

Comment:
While there is no requirement to put them in a specific place (as they are modals). I usually put them at the end of the component. 
For me it's easier to find them if I know that the modals are by the end - as they don't really share the visual place anywhere else in the DOM.